server/src/auth/service.js

```diff
@@ -0,0 +1,33 @@
+const UserModel = require('../user/model')
+const bcrypt = require('bcryptjs')
+const jwt = require('jwt-simple')
+const cfg = require('../../config/auth')
+
+exports.login = (req, res) => {
+  // check if the body contains an email and a password
+  if (req.body.email && req.body.password) {
+    // body assignments
```

Comment:
This comment is unclear and useless. Comments usually should answer the question `WHY` certain things are done and not `how` (developers see the assignation, there is no mystery :) )